### ISSUE TITLE
Change Single Column Header Background Color

### ISSUE THREAD

**Issue by la-colinares** [ISchwarz23/SortableTableView#185]
Hello, is there any possibilities to change each background color of a columns?
For example, 
Column A = Green Color
Column B = Red Color
Column C= Blue Color

Thank you and more power!

**Comment by ISchwarz23** [comment]
No, unfortunately the `TaeDataRowBackgroundProvider` will always set the same background for the complete row. The only thing you could do is setting the background color to your cell views inside your `TableDataAdapter`. But I guess the result will not be satisfying.
